src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1684,6 +1685,31 @@ More precisely, for a specific node provider, it is the sum of all time interval
 The accounting information can be accessed only through a *JMX client* or the ProActive Resource Manager *command line*.
 
 [[_run_as_me]]
+
+=== Metric to monitor Scheduler load
+
+*Needed Nodes* metric shows how much more nodes Scheduler needs to schedule all possible tasks,
```

Comment:
Same as above:
`Needed` - Metric shows how _many additional_ nodes _the_ Scheduler needs to _[dispatch|process]_ all _[pending|queued]_ tasks.